README.md

```diff
@@ -49,7 +49,8 @@ unless real values don't help users know what to change.
 ```hcl
 module "service_auth_cbr_rules" {
   # Replace "main" with a GIT release version to lock into a specific release
-  source                = "git::https://github.com/terraform-ibm-modules/terraform-ibm-s2s-auth.git?ref=main"
+  source                = "terraform-ibm-modules/terraform-ibm-s2s-auth/ibm"
```

Comment:
that source is wrong. hashicorp strips out the `terraform-ibm` prefix. So it should be `"terraform-ibm-modules/s2s-auth/ibm"`

Reply:
fixed